docs/03-pages/01-building-your-application/08-upgrading/03-version-13.mdx

```diff
@@ -72,6 +72,12 @@ To upgrade your links to Next.js 13, you can use the [`new-link` codemod](/docs/
 
 The behavior of [`next/script`](/docs/pages/api-reference/components/script) has been updated to support both `pages` and `app`. If incrementally adopting `app`, read the [upgrade guide](/docs/pages/building-your-application/upgrading).
 
+### `useRouter` Hook
```

Comment:
I believe this should be under the App Router part, not the general Next.13 part, since it only applies if you're using App Router.

Reply:
Moved to the "App Router Incremental Adoption Guide" instead.